src/commands/schema/cat.js

```diff
@@ -0,0 +1,46 @@
+const FaunaCommand = require("../../lib/fauna-command.js");
+const { Args } = require("@oclif/core");
+const fetch = require("node-fetch");
+
+class CatSchemaCommand extends FaunaCommand {
+  async run() {
+    const filename = this.args.filename;
+    const {
+      connectionOptions: { domain, port, scheme, secret },
+    } = await this.getClient();
+
+    try {
+      const res = await fetch(
+        `${scheme}://${domain}:${port}/schema/1/files/${filename}`,
```

Comment:
does this work if `domain` and `port` aren't set in the `~/.fauna-shell` file? I ran into a bug related to that with `eval --version 10`. It might be better to move this into a more general `endpoint` function on the client.

Reply:
I think it would be prudent to pull the duplicated URL generation logic out to a helper (at least the part which handles scheme, domain, and port.)

It also doesn't look like these commands use the endpoint configuration at all?

Reply:
followup: I see it's getting these from the client. That was not obvious (rusty JS here...). Seems like that at least needs to be explained or cleaned up, though.

Reply:
Sorry, is there a particular thing needed from this change?

Reply:
I'd appreciate if fetch setup were factored out into a shared helper (maybe a shared SchemaCommand superclass?). And testing to ensure that domain and port are either always set, or that the code can handle those not being present (I haven't looked at the internals, just re-raising @macmv's concern).